code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -322,12 +322,10 @@
 
 /obj/item/weapon/nullrod/carp/attack_self(mob/living/user)
 	if(used_blessing)
```

Comment:
Er, what? Why is this not just `if(!used_blessing && user.mind && user.mind.isholy)`